db/backup.go

```diff
@@ -237,7 +242,8 @@ func (db *db) basicExport(ctx context.Context, txn datastore.Txn, config *client
 							if err != nil {
 								return err
 							}
-							foreignDoc, err := foreignCol.Get(ctx, foreignDocID, false)
+							// TODO=ACP: Issue # 2430 - Add identity ability to export
```

Comment:
todo: Please add the whole link, else this line might be missed doing a Ctrl+f

Reply:
- [x] Full link